core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -836,21 +856,6 @@ private[spark] class SparkSubmit extends Logging {
     (childArgs.toSeq, childClasspath.toSeq, sparkConf, childMainClass)
   }
 
-  private def renameResourcesToLocalFS(resources: String, localResources: String): String = {
```

Comment:
I read the codes multiple times for sure, and I think this code does a duplicated job.
If I am not horribly wrong somewhere:
- `localResources` itself are always local files, and `resources` will always be replaced to `localResources`.
-  If `resources` is `null, then `localResources` will be `null` too

cc @skonto from https://github.com/apache/spark/commit/5e74570c8f5e7dfc1ca1c53c177827c5cea57bf1